backend/song/api/urls.py

```diff
@@ -0,0 +1,62 @@
+from django.urls import path, re_path
+
+from rest_framework import permissions
+
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+from backend.song.api.views import (
+    CategoriesList,
+    CategoriesDetail,
+    ArtistsList,
+    ArtistsDetail,
+    AlbumsList,
+    AlbumsDetail,
+    SongsList,
+    SongsDetail,
+    LanguagesList,
+    LanguagesDetail,
+    LyricsList,
+    LyricsDetail,
+    LyricRequestsList,
+    LyricRequestsDetail,
+    TranslationsList,
+    TranslationsDetail
+)
+
+app_name = 'song'
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Nzembo API",
+        default_version='v1',
+        description="Nzembo is a platform that allows people to share lyrics, translations of CD songs",
+        terms_of_service="https://www.google.com/policies/terms/",
+        contact=openapi.Contact(email="makutanolucien@gmail.com"),
+        license=openapi.License(name="BSD License"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
+
+urlpatterns = [
+    re_path(r'^swagger(?P<format>\.json|\.yaml)$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
+    re_path(r'^swagger/$', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
+    re_path(r'^redoc/$', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
```

Comment:
@tadomikikuto-bit, can we have this in the global `url.py` file so that we can have other endpoints in the documentation?